src/sql/azdata.d.ts

```diff
@@ -3340,7 +3342,7 @@ declare module 'azdata' {
 	/**
 	 * Editor properties for the editor component
 	 */
-	export interface EditorProperties extends ComponentProperties {
+	export interface EditorProperties extends EditorComponent {
```

Comment:
This should be the other way around generally - the component should have all the properties but not vice versa (since the component can have stuff like helper methods and whatnot)